cmd/adapter/adapter.go

```diff
@@ -53,6 +53,8 @@ type PrometheusAdapter struct {
 	AdapterConfigFile string
 	// MetricsRelistInterval is the interval at which to relist the set of available metrics
 	MetricsRelistInterval time.Duration
+	// MetricsStartDuration is the period to query available metrics for
+	MetricsStartDuration time.Duration
```

Comment:
nit: this name is not immediately obvious.  Perhaps `max-metrics-age` or something

Reply:
Renamed to `metrics-max-age` to be consistent with existing `metrics-relist-interval` flag name.